util/dvsim/nc_post_cmd.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+#ignore error 6, it is due to some misconfig of nc commands
+printf "Job $1 exited with code $2\n"
+if [ $2 -eq 0 ] || [ $2 -eq 6 ]
+then
+  printf "Successfully completed\n"
+  printf "Job $1 done\n"
+else
+  printf "Job $1 failed with error code $2\n"
+  #printf "check failed job info\n"
+  #nc info -l $1
+  #printf "check why the job failed\n"
+  #nc why $1
+fi
+cwd=$(realpath ..)
```

Comment:
Why the ".." here? At a quick glance, the working directory is `.`?

Reply:
done